crate/operator/main.py

```diff
@@ -360,25 +360,30 @@ async def cluster_create(
     )
 
     if "backups" in spec:
-        backup_metrics_labels = base_labels.copy()
-        backup_metrics_labels[LABEL_COMPONENT] = "backup"
-        backup_metrics_labels.update(meta.get("labels", {}))
-        kopf.register(
-            fn=subhandler_partial(
-                create_backups,
-                owner_references,
-                namespace,
-                name,
-                backup_metrics_labels,
-                http_port,
-                prometheus_port,
-                spec["backups"],
-                image_pull_secrets,
-                "ssl" in spec["cluster"],
-                logger,
-            ),
-            id="backup",
-        )
+        if config.CLUSTER_BACKUP_IMAGE is None:
+            logger.warning(
```

Comment:
I think the log level `WARN` is too high for this information, I'd rather use `INFO`.